### DISCUSSION TITLE
can anyone tell me that why this error ??

### DISCUSSION THREAD

**Discussion by dev-Crypto360** [Cyfrin/foundry-full-course-cu#868] [General]
![photo_2023-09-14_18-03-01](https://github.com/Cyfrin/foundry-full-course-f23/assets/80654518/95d7fc63-80f7-48dc-8127-d726c8e8dffd)

**Comment by ebok21** [comment]
You don't have any testnet ETH in your Metamask wallet. You can get some from this [faucet](https://sepoliafaucet.com/) if you're using the Sepolia Network.
NOTE: These are fake funds that only exist for the purpose of testing, they don't hold any real value!

**Comment by dev-Crypto360** [comment]
Thankyou ❤️
I already successfully made this actually problem was my solidity version,
I was Using Solidity version ^0.8.20;  For another Chain but you know it's
only support on Eth Chain & then I changed it to ^0.8.19; Then it was work
😊

**Comment by albin-george-kurian** [comment]
you have no ether in your account so try changing account and also try transact lesser amount than 100ETH at one time..
